src/config.rs

```diff
@@ -1456,15 +1456,17 @@ impl Config {
     #[cfg(target_os = "linux")]
     str_value!(linux, emerge_update_flags);
 
-    pub fn should_execute_remote(&self, remote: &str) -> bool {
-        if let Ok(hostname) = hostname() {
-            if remote == hostname {
+    pub fn should_execute_remote(&self, hostname: Result<String>, remote: &str) -> bool {
```

Comment:
I am curious why we added this new argument (`hostname`),  looks like the previous way how we handled it still works

Reply:
Ahh, I get it, it is for tests